horovod/spark/common/util.py

```diff
@@ -710,3 +710,33 @@ def _get_assigned_gpu_or_default(default):
 
 def is_databricks():
     return "DATABRICKS_RUNTIME_VERSION" in os.environ
+
+
+def get_spark_df_output_schema(input_df_schema, label_cols, output_cols):
+    if len(label_cols) != len(output_cols):
```

Comment:
@tgaddair Please check here. Is it OK to always assume label_cols and output_cols are 1:1 matching?